### ISSUE TITLE
Setting Admins within the control panel

### ISSUE THREAD

**Issue by JRWR** [mcpa-stlouis/hack-the-arch#44]
Right now to set an admin you have to do it within the database, this is not ideal as you might want more than one admin able to add challenges in or manage users / teams.

**Comment by paullj1** [comment]
Yeah, so this was somewhat of an intentional design choice... the point being that if the user was able to find a flaw in the app, the back end will reject all attempts to modify that field in the database.  I've gone back and forth on this myself.  Rails is secure enough, but there are always going to be holes.

**Comment by JRWR** [comment]
I would put a note in the README about this then if there isn't one already
